frontend/src/views/web/challenge/submission.html

```diff
@@ -20,18 +20,93 @@
+                <div class="col s11 m11" >
+                    <span class="fs-18">
+                        <strong>Instructions to Submit Docker Image using EvalAI-CLI</strong>
+                    </span>
+                    <ol>
+                        <li>Install EvalAI-CLI by using the command:<br>
+                            <code>$ pip install evalai
+                                <button class="btn ev-btn-dark waves-effect waves-dark grad-btn code-btn" ng-click="challenge.showConfirmation('Copied the Command')" ngclipboard data-clipboard-text="pip install evalai">
+                                    <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard."></i>
+                                </button>
+                            </code>
+                        </li>
+                        <li>Configure your CLI by using the commad:<br>
+                                <code>$ evalai set_token {{challenge.authCode}}
+                                    <button class="btn ev-btn-dark waves-effect waves-dark grad-btn code-btn" ng-click="challenge.showConfirmation('Copied the Command')" ngclipboard data-clipboard-text="evalai set_token {{challenge.authCode}}">
+                                        <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard."></i>
+                                    </button>
+                                    <button class="btn ev-btn-dark waves-effect waves-dark grad-btn code-btn" ng-click="challenge.showAuthCode()">
+                                        {{challenge.authCodeToggle}} Auth Token
+                                    </button>
+                                </code>
+                        </li>
+                        <li>
+                            For more details here are docs <a href="https://evalai-cli.cloudcv.org/" class="blue-text" target="_blank">EvalAI-CLI</a>
+                        </li>      
+                    </ol>
                 </div>
-                <div class="ev-md-container ev-panel-title"><strong>Please fill the following fields to submit</strong></div>
-                <div class="row">
+            </div>
+            <div class="ev-md-container ev-card-panel ev-z-depth-5 ev-challenge-view eval-card" ng-if="challenge.page.is_docker_based">
+                <span class="fs-18">
+                    <strong>Submission Limits</strong>
+                </span><br>
+                <span class="fs-14">
+                    <strong>Team Name:</strong> {{challenge.phaseRemainingSubmissions.participant_team}}
+                </span><br>
+                <span class="fs-14">
+                    <strong>Team ID:</strong> {{challenge.phaseRemainingSubmissions.participant_team_id}}
+                </span>
+                <table class="highlight centered">
+                    <thead>
+                        <tr>
+                            <th>Phase</th>
+                            <th>Start Date</th>
```

Comment:
Let's merge start date and end date in a single column. Something like: 
![screenshot 2019-02-16 at 8 11 38 pm](https://user-images.githubusercontent.com/12206047/52907033-15a8ad00-3227-11e9-95ac-0570d8f59f1b.png)